sources/29-web2py-english/07.markmin

```diff
@@ -763,7 +763,7 @@ db.define_table('person',
 db.define_table('dog',
     Field('owner', 'reference person'),
     Field('name', requires=IS_NOT_EMPTY()))
-db.dog.owner.requires = IS_IN_DB(db,db.person.id,'%(name)s')
+db.dog.owner.requires = IS_IN_DB(db, 'db.person.id', '%(name)s')
```

Comment:
I will take the PR but can you fix this. It should be without quotes or just 'person.id' in quotes.